setup/setup.py

```diff
@@ -39,9 +39,10 @@ def get_dirlist(_rootdir):
 
 setuptools.setup(
     name='nuscenes-devkit',
-    version='1.1.5',
+    version='1.1.6',
```

Comment:
Does this version number relate to the versions in the README.  If they should be the same, pls update there properly. tks.